series/series.go

```diff
@@ -647,3 +651,64 @@ type indexedElements []indexedElement
 func (e indexedElements) Len() int           { return len(e) }
 func (e indexedElements) Less(i, j int) bool { return e[i].element.Less(e[j].element) }
 func (e indexedElements) Swap(i, j int)      { e[i], e[j] = e[j], e[i] }
+
+// StdDev calculates the standard deviation of a series
+func (s Series) StdDev() float64 {
+	stdDev := stat.StdDev(s.Float(), nil)
+	return stdDev
+}
+
+// Mean calculates the average value of a series
+func (s Series) Mean() float64 {
+	stdDev := stat.Mean(s.Float(), nil)
+	return stdDev
+}
+
+// Max return the biggest element in the series
+func (s Series) Max() Element {
+	if s.elements.Len() == 0 {
+		return nil
+	}
+	max := s.elements.Elem(0)
+	for i := 0; i < s.elements.Len(); i++ {
+		elem := s.elements.Elem(i)
+		if elem.Greater(max) {
+			max = elem
+		}
+	}
+	return max
+}
+
+// Min return the lowest element in the series
+func (s Series) Min() Element {
+	if s.elements.Len() == 0 {
+		return nil
```

Comment:
My problem with not returning nil is that we would have to return a zero value for the underlying type, which would be a wrong answer. Like, you have a series of integers that's empty... both min and max would return 0, which is an incorrect answer. If I treat everybody as a float to return a NaN, I lose the capability of returning the min/max value for string and bool series. I was thinking about creating separate MaxStr and MinStr methods to handle strings and maybe treat bools as numerics. Then we would be able to use float64 for all numeric values... what do you thing about this approach?